sktalk/corpus/parsing/cha.py

```diff
@@ -44,3 +59,35 @@ def _clean_utterance(cls, utterance):
         utterance = re.sub(r" {2}", " ", utterance)
         utterance = re.sub(r"[ \t]{1,5}$", "", utterance)
         return utterance
+
+    @classmethod
+    def _extract_timing(cls, line):
+        timing = re.search(r"[0-9]{1,9}_[0-9]{1,9}", line)
+        try:
+            timing = timing.group()
+            timing = timing.split("_")
+            timing = [int(t) for t in timing]
```

Comment:
This might break something below.

```suggestion
        timing = re.search(r"([0-9]{1,9})_([0-9]{1,9})", line)
        try:
            timing = [int(timing.group(1)), int(timing.group(2))]
```

Reply:
I refactored to the point where I thought this would only make things more complex...